flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/DispatcherRestEndpoint.java

```diff
@@ -55,16 +59,19 @@
 public class DispatcherRestEndpoint extends RestServerEndpoint {
 
 	private final GatewayRetriever<DispatcherGateway> leaderRetriever;
+	private final Configuration clusterConfiguration;
 	private final RestHandlerConfiguration restConfiguration;
 	private final Executor executor;
 
 	public DispatcherRestEndpoint(
-			RestServerEndpointConfiguration configuration,
+			Configuration clusterConfiguration,
+			RestServerEndpointConfiguration endpointConfiguration,
```

Comment:
Shall we remove the order between `clusterConfiguration` and `endpointConfiguration`? I usually like to have the super class arguments first.

Reply:
Will change.